ingest/build-configs/nextstrain-automation/config.yaml

```diff
@@ -0,0 +1,30 @@
+# This configuration file should contain all required configuration parameters
+# for the ingest workflow to run with additional Nextstrain automation rules.
+
+# Custom rules to run as part of the Nextstrain automated workflow
+# The paths should be relative to the ingest directory.
+custom_rules:
+  - build-configs/nextstrain-automation/upload.smk
+
+# Nextstrain CloudFront domain to ensure that we invalidate CloudFront after the S3 uploads
+# This is required as long as we are using the AWS CLI for uploads
+cloudfront_domain: "data.nextstrain.org"
+
+# Nextstrain AWS S3 Bucket with pathogen prefix
+# Replace <pathogen> with the pathogen repo name.
+s3_dst: "s3://nextstrain-data/files/workflows/seasonal-cov"
+
+# Mapping of files to upload
+files_to_upload:
+  ncbi_229e.ndjson.zst: data/229e/ncbi.ndjson
+  ncbi_hku1.ndjson.zst: data/hku1/ncbi.ndjson
+  ncbi_nl63.ndjson.zst: data/nl63/ncbi.ndjson
+  ncbi_oc43.ndjson.zst: data/oc43/ncbi.ndjson
+  metadata_229e.tsv.zst: results/229e/metadata.tsv
+  sequences_229e.fasta.zst: results/229e/sequences.fasta
+  metadata_hku1.tsv.zst: results/hku1/metadata.tsv
+  sequences_hku1.fasta.zst: results/hku1/sequences.fasta
+  metadata_nl63.tsv.zst: results/nl63/metadata.tsv
+  sequences_nl63.fasta.zst: results/nl63/sequences.fasta
+  metadata_oc43.tsv.zst: results/oc43/metadata.tsv
+  sequences_oc43.fasta.zst: results/oc43/sequences.fasta
```

Comment:
_nit_

I slightly prefer a hierarchical file structure on S3 to match the Nextstrain [standard data files documentation](https://docs.nextstrain.org/en/latest/reference/data-files.html).

| current  | new |
| -------- | ---- |
| `ncbi_<virus>.ndjson/zst` | `<virus>/ncbi.ndjson.zst` |
| `metadata_<virus>.tsv.zst` | `<virus>/metadata.tsv.zst` |
| `sequences_<virus>.fasta.zst` | `<virus>/sequences.fasta.zst` |

I _think_ this will make surfacing these files on nextstrain.org easier in the long run.

Reply:
no objections — do I need to go in and manually clean up the files I've already uploaded to the bucket?

Reply:
> do I need to go in and manually clean up the files I've already uploaded to the bucket?

That would be much appreciated 🙏

Reply:
okay, cleaned them up.